README.md

```diff
@@ -66,20 +77,31 @@ Example `config.yaml` configuration file:
                       max: 100
                       step: 1
                       default: 20
+            nginx/backend:
+                settings:
+                    cpu:
+                        pinned: True # Optional
+                    mem:
+                        pinned: True # Optional
 ```
 
 To exclude a deployment from tuning, set `optune.ai/exclude` label to `'1'`. If you include the driver in the
 application namespace, please make sure you define this label in driver's Deployment object.
 
+The configuration may also define an `adjust_on` clause at the same level as the section `application`. When specified
+`adjust_on` should define a python statement to be used as a condition for enabling adjustment. This statement has access
```

Comment:
Might be better to include in one of the examples, i.e. `adjust_on: data["control"]["userdata"]["deploy_to"] == "canary"`

Reply:
Updated README with an example, moved adjust_on section to be above example